docs/technical_specs/model_libraries/power_generation/flowsheets/subcritical_power_plant.rst

```diff
@@ -0,0 +1,229 @@
+The process flow diagram is shown in Figure 1.  The entire process is modeled in two sub-flowsheets, one for the boiler system and the other for the steam cycle system, corresponding to two separate files named “boiler_subfs.py” and “steam_cycle_subfs.py”, respectively.  The main flowsheet contains the two sub-flowsheets in a file named “plant_dyn.py”.
+
+Figures Process Flow Diagram:
+
+.. figure:: subcr_pc_PFD.png
+  :width: 800
+  :align: center
+
+It can be seen from the figure that primary air is first split to two inlet streams, one goes through the primary air sector of the regenerative air preheater where it is heated, and the other, also known as tempering air, bypasses the air preheater. The two streams are then mixed and connected with the fire side of the boiler.  The coal stream is also fed to the fire side of the boiler. While this boiler sub-flowsheet does not contain a specific coal mill model, the partial vaporization of the moisture in the raw coal is modeled in the fire-side boiler model.  The secondary air enters the secondary air sector of the air preheater.  After being heated in the air preheater, the hot secondary air stream enters boiler’s windbox, from which it enters the furnace either as the secondary air of the burners or as overfire air.  The pulverized coal from primary air stream is eventually burned in the boiler by both primary and secondary air to form flue gas that leaves the boiler with small amount of unburned fuel in fly ash.  The hot flue gas then goes through the boiler backpass consisting of multiple convective heat exchangers including first the hot reheater, then the cold reheater, the primary superheater, and the economizer.  Finally, the flue gas enters the air preheater to heat the cold primary air and secondary air before entering the downstream equipment which is not modeled.  The feed water from steam cycle system enters the economizer to absorb the heat transferred from the flue gas and leaves the economizer at a temperature slightly below its saturation temperature.  The subcooled water then goes to the boiler drum through water pipes where it mixes with the saturated water separated from the water/steam mixture from the boiler waterwall.  The mixed water stream splits to two streams. A small amount of water leaves the system as a blowdown water and the main portion of the water stream goes through eight downcomers to enter the bottom of waterwall tubes.  The vertical waterwall is modeled by multiple waterwall section models in series. The subcooled water from the downcomers is heated by the combustion products inside the boiler and part of the liquid are vaporized, forming a liquid-vapor 2-phase mixture and eventually enters the drum to complete a circuit for natural circulation of the feed water, in which the density difference between the liquid in the downcomers and the 2-phase mixture in the waterwall tubes drives the circulating flow.
```

Comment:
This is a little painful, but you may want to actually break the lines of text rather than using word wrap.  It would make it easier to make review comments on specific lines.